diff --git a/public/wp-content/plugins/social-media-auto-publish/admin/publish.php b/public/wp-content/plugins/social-media-auto-publish/admin/publish.php
index fed2e6b..627b650 100755
--- a/public/wp-content/plugins/social-media-auto-publish/admin/publish.php
+++ b/public/wp-content/plugins/social-media-auto-publish/admin/publish.php
@@ -116,9 +116,8 @@ function xyz_link_publish($post_ID) {
     $lnaf=get_option('xyz_smap_lnaf');
 	
 	$postpp= get_post($post_ID);global $wpdb;
-	$entries0 = $wpdb->get_results( 'SELECT user_nicename FROM '.$wpdb->prefix.'users WHERE ID='.$postpp->post_author);
-	foreach( $entries0 as $entry ) {			
-		$user_nicename=$entry->user_nicename;}
+        $_ao_userdata = get_userdata($postpp->post_author);
+        $user_nicename = $_ao_userdata->display_name;
 	
 	if ($postpp->post_status == 'publish')
 	{
diff --git a/public/wp-content/plugins/social-media-auto-publish/social-media-auto-publish.php b/public/wp-content/plugins/social-media-auto-publish/social-media-auto-publish.php
index 094fbc2..9689481 100755
--- a/public/wp-content/plugins/social-media-auto-publish/social-media-auto-publish.php
+++ b/public/wp-content/plugins/social-media-auto-publish/social-media-auto-publish.php
@@ -70,15 +70,15 @@ require_once( dirname( __FILE__ ) . '/admin/metabox.php' );
 require_once( dirname( __FILE__ ) . '/admin/publish.php' );
 require_once( dirname( __FILE__ ) . '/admin/admin-notices.php' );
 
-if(get_option('xyz_credit_link')=="smap"){
-
-	add_action('wp_footer', 'xyz_smap_credit');
-
-}
-function xyz_smap_credit() {
-	$content = '<div style="clear:both;width:100%;text-align:center; font-size:11px; "><a target="_blank" title="Social Media Auto Publish" href="http://xyzscripts.com/wordpress-plugins/social-media-auto-publish/details" >Social Media Auto Publish</a> Powered By : <a target="_blank" title="PHP Scripts & Programs" href="http://www.xyzscripts.com" >XYZScripts.com</a></div>';
-	echo $content;
-}
+//if(get_option('xyz_credit_link')=="smap"){
+//
+//	add_action('wp_footer', 'xyz_smap_credit');
+//
+//}
+//function xyz_smap_credit() {
+//	$content = '<div style="clear:both;width:100%;text-align:center; font-size:11px; "><a target="_blank" title="Social Media Auto Publish" href="http://xyzscripts.com/wordpress-plugins/social-media-auto-publish/details" >Social Media Auto Publish</a> Powered By : <a target="_blank" title="PHP Scripts & Programs" href="http://www.xyzscripts.com" >XYZScripts.com</a></div>';
+//	echo $content;
+//}
 if(!function_exists('get_post_thumbnail_id'))
 	add_theme_support( 'post-thumbnails' );
 ?>
\ No newline at end of file
